src/operator/OrgOverlay.tsx

```diff
@@ -40,16 +40,20 @@ const OrgOverlay: FC = () => {
     limitsStatus,
     handleGetLimits,
     handleGetOrg,
+    handleReactivateOrg,
     handleUpdateLimits,
     organization,
     orgStatus,
+    reactivateOrgStatus,
     setLimits,
     updateLimitStatus,
   } = useContext(OverlayContext)
   const {hasWritePermissions} = useContext(OperatorContext)
 
   const {orgID} = useParams<{orgID: string}>()
   const history = useHistory()
+  const canReactivateOrg =
+    hasWritePermissions && organization?.state === 'suspended'
 
```

Comment:
Just FYI - I know this comes from the separate operator endpoint, so really separate issue. But we ended up renaming the state property on the frontend for clarity when it's returned in org objects in the array of orgs returned by the accounts/:accountId/orgs endpoint (we call it `provisioningStatus`). I don't have strong feelings though about whether it's worth changing here too.

https://github.com/influxdata/ui/pull/6343